src/triggers/errors.ts

```diff
@@ -7,7 +7,7 @@ export const errorRetry: Trigger<undefined> = async (clients) => {
   const errorQuery = `select *
   from "${Table.nftProcessErrors}"
   where ("numberOfRetries" < '${NUMBER_OF_RETRIES}' or "numberOfRetries" is null)
-  and (extract(minute from '${new Date().toISOString()}' - "lastRetry") >=1 or "lastRetry" is null)
+  and (age(now(), "lastRetry") >= make_interval(mins => cast(exp(coalesce("numberOfRetries", 0)) as int)) or "lastRetry" is null)
```

Comment:
looks like this does e^numberOfRetries, with e meaning Euler's number?

so the retry intervals will be:
1 min, 2.7 mins, 7.2, 19.6, 53 ?

This doesn't get us past 60mins which i think will be necessary for some cases (like sound.xyz early-access drops, where they have a private 1hour mint period, during which we'd error since we cant access the data, and after which we'd process correctly)

I guess in terms of requirements, the important cases to cover are:
 - first retry after 1 minute
 - next retry after 5 mins (to cover the cases where ipfs propagation is a little slow
 - retry after 61 mins (to cover early private drops use case)
 
maybe we can just use pow("numberOfRetries", 2.8), ie to the power of 2.8, this gets us:
1, 2.8, 7.84, , 21.95, 61.4

Reply:
Ah I was relying on us hitting 5 retries but I see we do `<` instead of `<=`.

Ok, so 2.8 doesn't get us where we want: { 1; 6,9; 21,6; 48,5; }

I'll adjust to use `pow("numberOfRetries", 3)`: { 1; 8; 27; 64; } 👍